package.json

```diff
@@ -56,17 +54,27 @@
     "eslint-plugin-import": "^2.29.1",
     "eslint-plugin-jsx-a11y": "^6.9.0",
     "eslint-plugin-md": "^1.0.19",
-    "eslint-plugin-react": "^7.34.3",
+    "eslint-plugin-react": "^7.35.0",
     "eslint-plugin-react-hooks": "^4.6.2",
-    "jsdom": "^24.1.0",
-    "nx": "18.3.5",
-    "prettier": "^3.3.2",
-    "storybook": "^8.1.11",
+    "jsdom": "^24.1.1",
+    "nx": "~18.3.5",
+    "prettier": "^3.3.3",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
+    "storybook": "^8.2.9",
     "ts-node": "^10.9.2",
     "tslib": "^2.6.3",
-    "typescript": "^5.5.3",
-    "vite": "^5.3.3",
+    "typescript": "^5.5.4",
+    "vite": "^5.4.1",
     "vite-plugin-dts": "^3.9.1",
     "vitest": "^1.6.0"
+  },
+  "peerDependencies": {
+    "@emotion/react": "^11.0.0",
```

Comment:
`peerDependencies` typically have different version constraints so this would be `"@emotion/react": ">=11.0.0",` assuming as much forward compatibility as possible.

Reply:
Yes, I have been frustrated by that in the past where a more restrictive version has been specified but it still works if you by-pass it. Given we can't know either way if a version will work it makes more sense to me to assume it will.

Reply:
Fair point.